__tests__/components/FriendRequestSidebarOptions/helpers.test.ts

```diff
@@ -190,55 +120,52 @@ describe('PusherClientHandler return tests', () => {
         client = new PusherClientHandler('stub_id', 0)
     })
 
-    test('return is a function, it should call unbind for friend requests',()=>{
-        const func = client.subscribeToPusher(jest.fn())
-        func()
-        expect(requestsUnbindSpy).toHaveBeenCalledWith('incoming_friend_requests', expect.anything())
-    })
+    //TODO: test here for "deny_friend"
 
-    test('return is a function, it should call unbind for new_friend',()=>{
+    test('Given that the return is a function: When that function is called, ' +
+        'then the new_friend event is unbound with the decrementCount callback',()=>{
```

Comment:
let's bite the bullet and make the entire statement for each of these descriptions on one line. If it's wordy, it may be a signal to break up the evaluation into more granular tests.